src/averageAges.js

```diff
@@ -14,12 +14,23 @@
  *
  * @return {number}
  */
+
+function calculate(array) {
+  return array.reduce((a, b) => a + b) / array.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const manAges = people.filter((person) => {
+    return (century
+      ? person.sex === 'm' && (Math.ceil(person.died / 100)) === century
+      : person.sex === 'm');
+  });
+
+  const manSumedAges = manAges.map((person1) => {
+    return person1.died - person1.born;
+  }, 0);
```

Comment:
```suggestion
  const men = people.filter((person) => {
    return (century
      ? person.sex === 'm' && (Math.ceil(person.died / 100)) === century
      : person.sex === 'm');
  });

  const menAges = men.map((person1) => {
    return person1.died - person1.born;
  }, 0);
```
Okay, now you changed your logic so it's not relevant